kernel/Kconfig

```
@@ -577,10 +577,20 @@ rsource "Kconfig.obj_core"
 menu "System Work Queue Options"
 config SYSTEM_WORKQUEUE_STACK_SIZE
 	int "System workqueue stack size"
-	default 4096 if COVERAGE_GCOV
-	default 2560 if WIFI_NM_WPA_SUPPLICANT
 	default 1024
 
+config SYSTEM_WORKQUEUE_STACK_SIZE_EXPLICIT
```

like heap, may be we can use same terminology?
```suggestion
config SYSTEM_WORKQUEUE_STACK_SIZE_IGNORE_MAX
```